administrator/components/com_admin/sql/updates/mysql/4.4.0-2023-10-31.sql

```diff
@@ -0,0 +1 @@
+UPDATE `#__modules` SET `access` = '2' WHERE `module` = 'mod_user' AND `client_id` = 1;
```

Comment:
```suggestion
UPDATE `#__modules` SET `access` = 2 WHERE `module` = 'mod_user' AND `client_id` = 1;
```
Don't use strings for integer values.